<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfWidget</class>
 <widget class="QWidget" name="ConfWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Threshold</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Range</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Standby Current</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="m_chkbDVDDEnabled">
        <property name="text">
         <string>DVDD</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="m_cbxRange_DVDD"/>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="m_spinUpperLimit_DVDD">
        <property name="maximum">
         <number>5500</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="m_chkbAVDDEnabled">
        <property name="text">
         <string>AVDD</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="m_cbxRange_AVDD"/>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="m_spinUpperLimit_AVDD">
        <property name="maximum">
         <number>5500</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="m_chkbDOVDDEnabled">
        <property name="text">
         <string>DOVDD</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="m_cbxRange_DOVDD"/>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="m_spinUpperLimit_DOVDD">
        <property name="maximum">
         <number>5500</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="m_chkbAFVCCEnabled">
        <property name="text">
         <string>AFVCC</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="m_cbxRange_AFVCC"/>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="m_spinUpperLimit_AFVCC">
        <property name="maximum">
         <number>5500</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="m_chkbVPPEnabled">
        <property name="text">
         <string>VPP</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="m_cbxRange_VPP"/>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="m_spinUpperLimit_VPP">
        <property name="maximum">
         <number>5500</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="m_chkbTotalEnabled">
        <property name="text">
         <string>Total</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="m_cbxRange_Total"/>
      </item>
      <item row="6" column="2">
       <widget class="QSpinBox" name="m_spinUpperLimit_Total">
        <property name="maximum">
         <number>50000</number>
        </property>
        <property name="value">
         <number>400</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="m_chkbContinueWhenFailed">
     <property name="text">
      <string>Continue when failed.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
